src/users/store/effects/users.effects.ts

```diff
@@ -20,18 +21,25 @@ export class UsersEffects {
       ofType(usersActions.LOAD_USERS),
       switchMap((action: any) => {
         return this.usersService.getListOfUsers(action.payload).pipe(
-          map((userDetails) => {
+          concatMap((userDetails) => {
             const amendedUsers = [];
+            let organisationProfileIds = [];
             userDetails.users.forEach((element) => {
               const fullName = `${element.firstName} ${element.lastName}`;
               const user = element;
               user.fullName = fullName;
               user.routerLink = `user/${user.userIdentifier}`;
               user.routerLinkTitle = `User details for ${fullName} with id ${user.userIdentifier}`;
               amendedUsers.push(user);
+              user.accessTypes = user.accessTypes || [];
+              organisationProfileIds = [...organisationProfileIds, ...user.accessTypes.map((accessType) => accessType.organisationProfileId)];
             });
 
-            return new usersActions.LoadUsersSuccess({ users: amendedUsers });
+            organisationProfileIds = [...new Set(organisationProfileIds)];
```

Comment:
A page refresh on the `/users`/ page does not have the organisation in the store so dispatching the event `orgActions.OrganisationUpdateUpdateProfileIds` will not be successful because there is no organisation to populate in the store